sklearn/model_selection/_search.py

```diff
@@ -525,6 +555,7 @@ def _fit(self, X, y, labels, parameter_iterable):
 
         estimator = self.estimator
         cv = check_cv(self.cv, y, classifier=is_classifier(estimator))
+        self.metric_name_ = _get_metric_name(self.scoring)
```

Comment:
I assume we will be renaming this attribute after you add multiple metric support. If it is necessary, you should prefix it with `_` rather than suffix